### ISSUE TITLE
Help - confused about "Filter" option

### ISSUE THREAD

**Issue by corobin** [bewisse/modheader#76]
Hi all,

Just want to clarify how ModHeader works.

The help page says to apply a modification to a specific website, add a "Filter" option

The filter option only has 1 field, for example, for a URL pattern it only has 1 box to type in the URL pattern but nothing to set what header to modify for it

1. Is the filter applying to all header modifications in that profile?

For example, if I want to apply 1 modification to all websites, and 1 modification to a specific website, is the correct way to do it to create 2 profiles. the 1st profile has the universal modification with no filter, and the 2nd profile has the specific modification rule, and a filter rule?

2. How are multiple filters applied? are they ORed or ANDed?

in other words, if for example I have two URL Pattern filters, then do the modifications apply to either website?

if i have two Exclude URL Pattern filters, then do the modifications apply to every website except for either of the two specified?

Thanks!

**Comment by hao1300** [comment]
> 1. Is the filter applying to all header modifications in that profile?
Yes

> 2. How are multiple filters applied? are they ORed or ANDed?
They are ANDed

The correct way is to create 2 profiles that are always enabled (right click on the profile icon to select that).

> if i have two Exclude URL Pattern filters, then do the modifications apply to every website except for either of the two specified?
Unfortunately, that will not work, because the filters are ANDed, so you will satisfy one condition but fails another. The best way to achieve this is with a more complex RegEx URL Pattern to satisfy everything except the two websites you want.

**Comment by corobin** [comment]
thanks for the explanation!
